wire/Makefile

```diff
@@ -50,11 +50,16 @@ wire/extracted_onion_experimental_csv:
 	@touch $@
 
 ifeq ($(EXPERIMENTAL_FEATURES),1)
-wire/gen_peer_wire_csv: wire/extracted_peer_experimental_csv wire/extracted_peer_wire_csv
-	@patch --silent -o $@ wire/extracted_peer_wire_csv $<
+EXPERIMENTAL_PEER_PATCHES := $(wildcard wire/extracted_peer_experimental_*)
+EXPERIMENTAL_ONION_PATCHES := $(wildcard wire/extracted_onion_experimental_*)
 
-wire/gen_onion_wire_csv: wire/extracted_onion_experimental_csv wire/extracted_onion_wire_csv
-	@patch --silent -o $@ wire/extracted_onion_wire_csv $<
+wire/gen_peer_wire_csv: wire/extracted_peer_wire_csv
+	@cp $< $@
+	@for exp in $(EXPERIMENTAL_PEER_PATCHES); do patch --silent $@ $$exp; done
```

Comment:
Let's do this via a temporary in case things go wrong?  Plus set -e to abort?

```
@set -e; trap "rm -f $@.$$$$" 0; cp $< $@.$$$$; for exp in $(EXPERIMENTAL_PEER_PATCHES); do patch --silent $@.$$$$ $$exp; done; mv $@.$$$$ $@
```